packages/terra-site/src/Index.jsx

```diff
@@ -121,6 +122,7 @@ ReactDOM.render((
       <Route path="time-input" component={TimeInputExamples} />
       <Route path="toggle" component={ToggleExamples} />
       <Route path="toggle-button" component={ToggleButtonExamples} />
+      {FormIndexRoutes}
     </Route>
```

Comment:
[This issue](https://github.com/cerner/terra-core/issues/697) was logged to handle just that.